tuxemon/core/components/player.py

```diff
@@ -453,7 +453,7 @@ def get_collision_dict(self, game):
         npc_positions = set()
 
         # Get all the NPC's tile monsters_in_play so we can check for collisions.
-        for npc in game.npcs:
+        for npc in game.npcs.items():

```

Comment:
https://docs.python.org/2/library/stdtypes.html#dict.items

Items return key, value pairs.  Maybe you meant?:
```python
for slug, npc in game.npcs.items():
   ...
```